otel-agent/k8s-helm/templates/scc.yaml

```diff
@@ -0,0 +1,49 @@
+{{/*
+Default values for SecurityContextConstraints
+*/}}
+
+{{- define "opentelemetry-coralogix.defaultSecurityContextConstraints" -}}
+priority: 10
+allowHostNetwork: true
+allowHostPorts: true
+allowHostPID: true
+allowHostDirVolumePlugin: true
```

Comment:
Do we know if all of these need to be allowed? Or could we at least put a comment why is each needed? AFAIK we need host network and host dir, but since I'm unsure what each one does, I cannot tell if we really need any other besides the two.